spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java

```diff
@@ -81,23 +80,29 @@
  */
 public class CreateChangelogViewProcedure extends BaseProcedure {
 
+  private static final ProcedureParameter TABLE_PARAM =
```

Comment:
I picked `CreateChangelogViewProcedure` as the first procedure to migrate because the params are diverse: map, array, string, boolean. If we go with this change, I'll update all procedures.